src/main/java/com/wdtinc/mapbox_vector_tile/adapt/jts/JtsAdapter.java

```diff
@@ -441,9 +431,6 @@ public static TileGeomResult createTileGeom(List<Geometry> g,
 
         // Guard: empty geometry coordinates
         final Coordinate[] geomCoords = geom.getCoordinates();
-        if(geomCoords.length <= 0) {
-            Collections.emptyList();
```

Comment:
I think instead of removing emptyList() call, can we just define our own 'emptyList()' convenience function in a utility class that returns a unmodifiable static list value that is empty?

Reply:
the code looks like dead code to me - there is no assignment from the empty list https://docs.oracle.com/javase/7/docs/api/java/util/Collections.html#emptyList()

Reply:
I think this is meant to return Collections.emptyList(), but the return statement is missing. Could be a bug discovery rather than dead code discovery.

Reply:
ok, I'll add the return statements.